src/components/App.js

```diff
@@ -63,48 +63,60 @@ const App = () => {
                 console.error(error);
             });
     }
-   
+
     const loadLabel = async () => {
-        const response = await github.get('labels');
+        const response = await github.get('/vue/labels');
```

Comment:
```suggestion
        const response = await github.get('/vue/labels');
```
Should be selected repo right?

Reply:
ay tama diay sir. ang g apil ra nako sir issues.